ignite/pkg/cosmosgen/template.go

```diff
@@ -78,6 +80,58 @@ func (t templateWriter) Write(destDir, protoPath string, data interface{}) error
 
 			return "./types/" + rel
 		},
+		"transformPath": func(path string) string {
```

Comment:
Maybe it's better to create a separate function to do it and create tests for it